Neos.Neos/Documentation/References/NeosFusionReference.rst

```diff
@@ -200,7 +200,8 @@ Render each item in ``items`` using ``itemRenderer`` and return the result as an
 :itemName: (string, defaults to ``item``) Context variable name for each item
 :itemKey: (string, defaults to ``itemKey``) Context variable name for each item key, when working with array
 :iterationName: (string, defaults to ``iterator``) A context variable with iteration information will be available under the given name: ``index`` (zero-based), ``cycle`` (1-based), ``isFirst``, ``isLast``
-:itemRenderer: (mixed, **required**) The renderer definition (simple value, expression or object) will be called once for every collection element (if ``itemRenderer`` cannot be rendered the path ``content`` is used as fallback for convenience in afx)
+:itemRenderer: (mixed, **required**) The renderer definition (simple value, expression or object) will be called once for every collection element to render the item (if ``itemRenderer`` cannot be rendered the path ``content`` is used as fallback for convenience in afx)
+:keyRenderer: (mixed, **optional**) The renderer definition (simple value, expression or object) will be called once for every collection element to render the key in the result collection (if ``itemRenderer`` cannot be rendered the path ``content`` is used as fallback for convenience in afx)
```

Comment:
The fallback to content part does not belong to the keyRenderer

Reply:
Thanks @mficzel, fixed.